alacritty/src/event.rs

```diff
@@ -286,9 +287,8 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
 
     fn start_selection(&mut self, ty: SelectionType, point: Point, side: Side) {
         self.terminal.selection = Some(Selection::new(ty, point, side));
-        *self.dirty = true;
-
         self.copy_selection(ClipboardType::Selection);
+        *self.dirty = true;
```

Comment:
Why was this moved? I actually kinda like putting it under the statement that actually requires the dirty flag.

The `copy_selection` doesn't care about dirty or not, right? So why not have the statement closer to what actually caused the dirty change?

Reply:
I've sorted them by length.

Reply:
Yeah I'm not sure that's the best choice here, especially since there's a newline in between?